inc/saiport.h

```diff
@@ -2272,6 +2272,18 @@ typedef enum _sai_port_attr_t
      */
     SAI_PORT_ATTR_ECMP_HASH_OFFSET,
 
+    /**
+     * @brief Operational FEC mode
+     *
+     * If port is down, the returned value should be SAI_PORT_FEC_MODE_NONE.
```

Comment:
also add the behavior when auto-neg is in progress

Reply:
@MaksymPry : please update this and then we can merge this PR.

Reply:
@abdosi Done, thanks